omnigibson/sensors/vision_sensor.py

```diff
@@ -466,7 +466,7 @@ def _obs_space_mapping(self):
             normal=((self.image_height, self.image_width, 3), -1.0, 1.0, np.float32),
             seg_semantic=((self.image_height, self.image_width), 0, MAX_CLASS_COUNT, np.uint32),
             seg_instance=((self.image_height, self.image_width), 0, MAX_INSTANCE_COUNT, np.uint32),
-            flow=((self.image_height, self.image_width, 3), -np.inf, np.inf, np.float32),
+            flow=((self.image_height, self.image_width, 4), -np.inf, np.inf, np.float32),
```

Comment:
why 4 dimensional?

Reply:
no idea -- that's just what we get from omni, go figure :0